content/en/platform/corda/1.6/cenm/release-notes.md

```diff
@@ -22,6 +22,12 @@ title: Release notes
 * Two network parameters for ledger recovery with Corda 4.11 have been added:
   * `recoveryMaximumBackupInterval`
   * `confidentialIdentityMinimumBackupInterval`
+* CENM now supports JDK Azul 8u392.
```

Comment:
I think keep this same style as from 1.5.9 release notes:

> The JDK version used by CENM has been upgraded from JDK 8.0.382 to JDK 8.0.392.

Reply:
Updated